tests/test_data/non_hrv_shell.zarr/.zattrs

```diff
@@ -0,0 +1,264 @@
+{
```

Comment:
always an interesting one, where we add a zarr file or we make it on the fly like[here](https://github.com/openclimatefix/ocf-data-sampler/blob/main/tests/conftest.py#L81). Perhaps good to leave it how it is and make an issue, to change this to be made on the fly?

Reply:
Sounds good, for satellite data I wasn't sure if any of the attributes or other metadata mattered or not so I followed the pattern used in the UK app for now